recipes/r-adehabitathr/bld.bat

```diff
@@ -1,2 +1,2 @@
-"%R%" CMD INSTALL --build . %R_ARGS%
-IF %ERRORLEVEL% NEQ 0 exit /B 1
+"%R%" CMD INSTALL --build . %R_ARGS%
+IF %ERRORLEVEL% NEQ 0 exit /B 1
```

Comment:
I'm not sure these files are from your PR, right? Do you mind rebasing or, if that is to ocomplicated, send a new "clean" PR with only your package?

Reply:
OK, thanks. I'll send a new one.